### ISSUE TITLE
Tessellation : PN triangles

### ISSUE THREAD

**Issue by MeinMein** [andyrst/ue4_metaballs#2]
Can you set the metaballs material to PN triangles tessellation without artifacts?
For me the vertices are all over the place.

**Comment by andyrst** [comment]
What do you mean "artifacts"? I can set any texture. But if you need some uv mapping like planar / box you need recalculate all uv data.

![image](https://cloud.githubusercontent.com/assets/9151008/14655571/96470ca6-068b-11e6-8fe5-90376470f53c.png)

**Comment by MeinMein** [comment]
Did you try enabling PN triangles tessellation on the material?
https://docs.unrealengine.com/latest/INT/Resources/ContentExamples/MaterialProperties/1_8/index.html
I don't want to spoil the surprise :sunglasses:

**Comment by andyrst** [comment]
No. i don't play with PN tesselation. Metaballs has "Grid steps" setting for control detail level (like tesselation multilpier).

**Comment by MeinMein** [comment]
I know, but I am trying to use the tessellation to offload part of the computation to the gpu.
As in: generate a low poly mesh and let the gpu smooth it out via PN triangles.

Did it work for you? Or did your triangles also go all over the place?

**Comment by andyrst** [comment]
MB using procedural mesh generation. I don't do any experements with PN triangles.

**Comment by MeinMein** [comment]
Ok, sorry to ask, but could you try it on your system?
Just pick PN Triangles in the material details tessellation drop-down:
https://answers.unrealengine.com/storage/temp/70544-tess01.png

**Comment by andyrst** [comment]
Flat tesselation working well. Currently i dont know how to fix issue with PN tesselation

**Comment by MeinMein** [comment]
Thanks, good to know the problem also exists on other systems.
I'll see whether I can fix it.

**Comment by andyrst** [comment]
Thanks. It would be nice if you can fix this issue.
